.github/workflows/test.yml

```diff
@@ -9,18 +9,47 @@ on:
 
 jobs:
   test:
-    runs-on: ${{ matrix.os }}
     timeout-minutes: 30
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-latest, macOS-latest, windows-latest]
-        nim: [1.6.20, 2.0.8]
+        platform:
+          - os: linux
+            runner: ubuntu-latest
+            cpu: amd64
+            shell: bash
+          - os: linux
+            runner: ubuntu-latest
+            cpu: i386
+            shell: bash
+          - os: macos
+            runner: macos-latest
+            cpu: amd64
+            shell: bash
+          - os: windows
+            runner: windows-2022
```

Comment:
Just out of curiosity, why not used `windows-latest`

Reply:
It was a mistake, fixed.